.github/workflows/deploy-fse-to-org.yml

```diff
@@ -0,0 +1,32 @@
+name: Deploy FSE plugin to WordPress.org
+
+# Until we are ready to move to continuous deployment to Simple and Atomic sites
+# we need a manual way to trigger this workflow. One way to do it is to push a tag
+# to wp-calypso repository. Since we already have other tags for desktop releases
+# I'm prefixing this matcher with fse so it doesn't run when it's not supposed to.
+# Once we are ready to move to continuous workflow we can remove this and trigger
+# the workflow on on each merge to master that contains changes in FSE plugin folder.
```

Comment:
We might still need tags:

https://github.com/10up/action-wordpress-plugin-deploy/blob/98923950204e68db278a958aa8b71e3b07904140/entrypoint.sh#L31

Reply:
At that point I was thinking we could roll our own script and automate that part too.